superset/assets/javascripts/explorev2/components/ChartContainer.jsx

```diff
@@ -186,7 +186,6 @@ class ChartContainer extends React.Component {
           id={this.props.containerId}
           ref={(ref) => { this.chartContainerRef = ref; }}
           className={this.props.viz_type}
-          style={{ overflowX: 'auto' }}
```

Comment:
this was added because there are some charts that don't scale to the width of the container, and so a scrollbar was needed to see the rest of the chart. i thought i did see this functionality in v1. i will try and find an example.